### ISSUE TITLE
Add a verification step after downloading issue for BM UPI deployment 

### ISSUE THREAD

**Issue by prsurve** [red-hat-storage/ocs-ci#4147]
From the ocs-ci looks it looks like the download of files worked but actually, file was not fully downloaded and that cause install to fail

Apr 19 04:09:44 magna012 dnsmasq-tftp[2012849]: file /var/lib/tftpboot/ocs4qe/baremetal/rhcos-installer-kernel-x86_64 not found

**Comment by petr-balogh** [comment]
Might be related to PSI outage - there are a lot of network connectivity issues

**Comment by prsurve** [comment]
> Might be related to PSI outage - there are a lot of network connectivity issues

No, it's not related to the PSI outage because we are downloading UPI related files from https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/ download might have failed because of n/w issue

**Comment by petr-balogh** [comment]
So yes, it can be then related if it failed cause of n/w issue as PSI is experiencing a lot of network issues. That's what I am trying to say. We even have problems to connect to the NFS share.   Is the download happening on jenkins VM (where ocs-ci is running) or from node?
